src/tests/ftest/util/general_utils.py

```diff
@@ -304,3 +312,802 @@ def verify_rebuild(pool, log, to_be_rebuilt, object_qty, record_qty, errors=0):
                 "Unexpected {} value: expected={}, detected={}".format(
                     key, expected, detected))
     return messages
+
+
+def check_pool_files(log, hosts, uuid):
+    """Check if pool files exist on the specified list of hosts.
+
+    Args:
+        log (logging): logging object used to display messages
+        hosts (list): list of hosts
+        uuid (str): uuid file name to look for in /mnt/daos.
+
+    Returns:
+        bool: True if the files for this pool exist on each host; False
+            otherwise
+
+    """
+    file_list = (uuid, "superblock")
+    expect = len(file_list) * len(hosts)
+    actual = 0
+    nodeset = NodeSet.fromlist(hosts)
+    task = task_self()
+
+    log.info("Checking for pool data on %s", nodeset)
+    for file in file_list:
+        task.run(
+            "test -e /mnt/daos/{}; echo $?".format(file), nodes=nodeset)
+        for buffer, node_list in task.iter_buffers():
```

Comment:
(pylint-redefined-builtin) Redefining built-in 'buffer'

Reply:
Fixed